packages/bundler/src/bundler.ts

```diff
@@ -181,7 +184,13 @@ async function createRollupConfig(definition: TypeSpecBundleDefinition): Promise
       }),
       (commonjs as any)(),
       (json as any)(),
+      (alias as any)({
```

Comment:
a bit unfortunate we have to do that but at least its only for openapi3 right now